log/log_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package log
+
+import (
+	"testing"
+
+	"github.com/sirupsen/logrus"
+)
+
+func TestNewLogger(t *testing.T) {
+	t.Parallel()
+
+	logrusLog := logrus.New()
+	logrusLog.SetLevel(logrus.TraceLevel)
+
+	tests := []struct {
+		name     string
+		logLevel Level
+	}{
+		{
+			name:     "debug",
+			logLevel: DebugLevel,
+		},
+		{
+			name:     "info",
+			logLevel: InfoLevel,
+		},
+		{
+			name:     "warn",
+			logLevel: WarnLevel,
+		},
+		{
+			name:     "error",
+			logLevel: ErrorLevel,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
missing a `tt := tt` for loop shadowing I believe. 

I'm surprised the linter isn't flagging this, but perhaps because the `t.Run` isn't using `t.Parallel`. Any reason not to?